lib/matplotlib/backend_managers.py

```diff
@@ -74,6 +73,43 @@ def __init__(self, canvas):
         self.keypresslock = widgets.LockDraw()
         self.messagelock = widgets.LockDraw()
 
+        if figure:
+            self.figure = figure
+
+    @property
+    def canvas(self):
+        """Canvas managed by FigureManager"""
+        return self._figure.canvas
+
+    @property
+    def figure(self):
+        """Figure that holds the canvas"""
+        return self._figure
+
+    @figure.setter
+    def figure(self, figure):
+        self.set_figure(figure)
+
+    def set_figure(self, figure, update_tools=True):
+        """
+        Set the figure that interact with tools
```

Comment:
Isn't this statement a bit backwards? That is, the tools interact with the figure, and not the other way around, as specified here?